examples/main.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_version = ">= 0.14.0"
+}
+
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "event_mapping" {
+  source             = "../"
+  bus_name = "the-night-bus"
```

Comment:
Excuse me sir I think you might have meant "knight bus" - https://harrypotter.fandom.com/wiki/Knight_Bus

🤣 😉